src/navigation/components/NavigationItem/NavigationItem.tsx

```diff
@@ -19,6 +19,9 @@ import './NavigationItem.scss';
 
 const b = block('navigation-item');
 
+// discuss how to be with this component
+// const BLOCK_NAME = 'navigation';
+
```

Comment:
we don't have blocks in navigation so here you need to create context manually

Reply:
ok